src/App.tsx

```diff
@@ -2,13 +2,17 @@ import React from 'react';
 import logo from './logo.svg';
 import { Route, Switch, BrowserRouter as Router } from 'react-router-dom';
 import './App.css';
-import {Home} from './Pages/Home';
+import {Home} from './Pages/Home/Home';
```

Comment:
1. Name like pages is better, only components name should starts with capitalize letter (lets add this as rule)
2.  Create pages/index.ts and export all components from pages folder. Than you will able to exports like 
`import {PageA, PageB} from './pages' `